.travis_scripts/mvnrepo.sh

```diff
@@ -12,6 +12,7 @@ if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; th
         rm target/mvn-repo/org/corfudb/corfu/${PROJECT_VERSION}/*.deb.md5
         rm target/mvn-repo/org/corfudb/corfu/${PROJECT_VERSION}/*.deb.sha1
         cp -R target/mvn-repo $HOME/mvn-repo-current
+        cp -R runtime/target/mvn-repo $HOME/mvn-repo-current
```

Comment:
honestly, I don't know how to check that this is the right fix until we actually run and upload to the Corfu-Repos repo. 

So by syntax only, I want to ask
1. should runtime/target be copied to the root of remote mvn-repo or to a remote runtime subdir?
2. should debian artifacts be removed also from under local runtime/target?